src/main/java/net/patchworkmc/manifest/accesstransformer/v2/ForgeAccessTransformer.java

```diff
@@ -50,12 +50,19 @@ public static ForgeAccessTransformer parse(Path path) throws ManifestParseExcept
 
 			for (String line : lines) {
 				// List of all words in this line with comments stripped
-				String[] words = line.split("#")[0].split(" ");
+				int commentIndex = line.indexOf('#');
+				String strippedLine = "";
 
-				if (words.length == 0) {
+				if (commentIndex != -1) {
+					strippedLine = line.substring(0, line.indexOf('#'));
+				}
```

Comment:
what if there is no comment? then strippedLine will be empty

Reply:
done